<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pivmake-gui.qrc">
    <normaloff>:/myimages/share/pivmake-gui/icons/pivmake-gui.png</normaloff>:/myimages/share/pivmake-gui/icons/pivmake-gui.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="icon">
        <iconset resource="../pivmake-gui.qrc">
         <normaloff>:/myimages/share/pivmake-gui/icons/if_polaroids_1055003.png</normaloff>:/myimages/share/pivmake-gui/icons/if_polaroids_1055003.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Work</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Step 0: Select the input files&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="3">
             <widget class="QPushButton" name="pushButton_create_listfilesdat">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a file list data file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Create</string>
              </property>
              <property name="icon">
               <iconset resource="../pivmake-gui.qrc">
                <normaloff>:/myimages/share/pivmake-gui/icons/if_compose_1055085.png</normaloff>:/myimages/share/pivmake-gui/icons/if_compose_1055085.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_listfilesdat">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This field contain the path of a text file with the list of images. Inside this file exist one image file path by line. &lt;/p&gt;&lt;p&gt;It is strongly recommended that the text file with the list of images has a filename extension &lt;span style=&quot; font-weight:600;&quot;&gt;*.listfiles&lt;/span&gt;. By example &lt;span style=&quot; font-weight:600;&quot;&gt;listfilesdat.listfiles&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Image files list:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_choose_listfilesdat">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose a file list data file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Choose</string>
              </property>
              <property name="icon">
               <iconset resource="../pivmake-gui.qrc">
                <normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Regions list:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_listregionsdat">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This field contain the path of text file with the list of regions. Inside this file is defined one region by line. &lt;/p&gt;&lt;p&gt;It is strongly recommended that the text file with the list of regions has a filename extension &lt;span style=&quot; font-weight:600;&quot;&gt;*.listregions&lt;/span&gt;. By example &lt;span style=&quot; font-weight:600;&quot;&gt;listregionsdat.listregions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_choose_listregionsdat">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose a region list data file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Choose</string>
              </property>
              <property name="icon">
               <iconset resource="../pivmake-gui.qrc">
                <normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButton_create_listregionsdat">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a region list data file.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Create</string>
              </property>
              <property name="icon">
               <iconset resource="../pivmake-gui.qrc">
                <normaloff>:/myimages/share/pivmake-gui/icons/if_compose_1055085.png</normaloff>:/myimages/share/pivmake-gui/icons/if_compose_1055085.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Step 1: Select the output files location&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_output_filename_pattern">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a name=&quot;result_box&quot;/&gt;This field contains the pattern of the output files names. If the pattern is &lt;span style=&quot; font-weight:600;&quot;&gt;listregionsdata&lt;/span&gt;, then the files will have the names:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;listregionsdata&lt;/span&gt;1.listregions&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;listregionsdata&lt;/span&gt;2.listregions&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;listregionsdata&lt;/span&gt;3.listregions&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>listregionsdata</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="checkBox_enable_filename_pattern">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable the selection of output filename pattern.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_output_directory">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The path where will be located all the output region list files. If the directory not exist, then it is created in the generate process.&lt;/p&gt;&lt;p&gt;&lt;a name=&quot;result_box&quot;/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;I&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;t is strongly recommended that this directory be a subdirectory of one of the * .litfiles or * .listregions files (&lt;/span&gt;&lt;a name=&quot;result_box&quot;/&gt;just to keep an order&lt;span style=&quot; font-weight:600;&quot;&gt;). &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_choose_output_directory">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose a output directory.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Choose</string>
              </property>
              <property name="icon">
               <iconset resource="../pivmake-gui.qrc">
                <normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Output filename:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Output directory:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Step 2: Generate the output files&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_generate_outfiles">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click here to tracking the regions and generate the output files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset resource="../pivmake-gui.qrc">
         <normaloff>:/myimages/share/pivmake-gui/icons/if_tools_1054957.png</normaloff>:/myimages/share/pivmake-gui/icons/if_tools_1054957.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Color of region:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_program_listfiles">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The program filename &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;listfiles&amp;quot; &lt;/span&gt;or &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;listfiles.exe&amp;quot;&lt;/span&gt; if It is in the system path. In other cases should be wrote the complete path of the program.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_find_program_listfiles">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Find and select the &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;listfiles&amp;quot;&lt;/span&gt; or &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;listfiles.exe&amp;quot;&lt;/span&gt; program.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Find</string>
            </property>
            <property name="icon">
             <iconset resource="../pivmake-gui.qrc">
              <normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_program_listregions">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The program filename &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;listregions&amp;quot; &lt;/span&gt;or &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;listregions.exe&amp;quot;&lt;/span&gt; if It is in the system path. In other cases should be wrote the complete path of the program.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>ListFiles program:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>ListRegions program:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Programs</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_find_program_listregions">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Find and select the &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;listregions&amp;quot;&lt;/span&gt; or &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;listregions.exe&amp;quot;&lt;/span&gt; program.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Find</string>
            </property>
            <property name="icon">
             <iconset resource="../pivmake-gui.qrc">
              <normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</normaloff>:/myimages/share/pivmake-gui/icons/folder-saved-search.png</iconset>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Output image type:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Output</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="comboBox_output_image_type">
            <item>
             <property name="text">
              <string>PNG - Portable Network Graphics</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>BMP - Bits Maps Protocole</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButton_color_region">
            <property name="text">
             <string>(255,0,0)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../pivmake-gui.qrc">
     <normaloff>:/myimages/share/pivmake-gui/icons/Information_icon.png</normaloff>:/myimages/share/pivmake-gui/icons/Information_icon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="../pivmake-gui.qrc">
     <normaloff>:/myimages/share/pivmake-gui/icons/Information_icon.png</normaloff>:/myimages/share/pivmake-gui/icons/Information_icon.png</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Documentation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../pivmake-gui.qrc"/>
 </resources>
 <connections/>
</ui>
